cdap-explore-jdbc/src/test/java/co/cask/cdap/explore/jdbc/ExplorePreparedStatementTest.java

```diff
@@ -45,9 +45,9 @@ public void executeTest() throws Exception {
                       (List<QueryResult>) Lists.<QueryResult>newArrayList())
     );
 
-
+    String ns = "ns1";
```

Comment:
The current identifier validation allowed empty string.
This PR changed it to now allow empty strings.